### PR TITLE
Uno R4 WiFi OTA update

### PR DESCRIPTION
**Pull request by pennam**
Apply after #99 
Needs WiFi firmware 0.3.0 -> https://github.com/arduino/uno-r4-wifi-usb-bridge/releases/tag/0.3.0-rc1

### CODE REVIEW

**Comment by JAndrassy** [comment]
Uno R4 **WiFi** OTA

File: libraries/OTAUpdate/examples/WiFiFirmwareOTA/WiFiFirmwareOTA.ino [hunk 1]
```diff
@@ -0,0 +1,127 @@
+/*
+  OTA
+
+  This sketch demonstrates how to make an self OTA Update the UNO R4 WiFi Firmware
```
Comment (per1234) [inline libraries/OTAUpdate/examples/WiFiFirmwareOTA/WiFiFirmwareOTA.ino:4]:
```suggestion
  This sketch demonstrates how to make the UNO R4 WiFi self-update its Wi-Fi module firmware via OTA
```

The previous wording was incorrect.

My intent was for my suggested text to have the same meaning as what was intended by the former text, but the meaning of the former text was difficult to decipher so it is possible I inadvertently shifted the meaning.

File: libraries/OTAUpdate/examples/WiFiFirmwareOTA/WiFiFirmwareOTA.ino [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    // don't continue
+    while (true);
+  }
+
+  String fv = WiFi.firmwareVersion();
+  if (fv < WIFI_FIRMWARE_LATEST_VERSION) {
+    Serial.println("Please upgrade the firmware");
+  }
+
+  Serial.print("Current WIFi firmware version: ");
```
Comment (per1234) [inline libraries/OTAUpdate/examples/WiFiFirmwareOTA/WiFiFirmwareOTA.ino:45]:
```suggestion
  Serial.print("Current WiFi firmware version: ");
```
Fix case of letter.

The correct spelling is actually the hyphenated "[Wi-Fi](https://en.wikipedia.org/wiki/Wi-Fi)". This spelling is not used in Arduino product names due to being trademarked, but in my opinion the correct spelling should be used in prose such as this comment where there should be no problem with trademark infringement. But I'll settle for just changing the letter case.

**Review by per1234 — CHANGES_REQUESTED** [review]

**Comment by pennam** [comment]
[ci](https://github.com/arduino/ArduinoCore-renesas/actions/runs/5940093056/job/16107924184?pr=100) is failing because library is not included in the previous core releases.

**Pull request merged**